src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,184 @@
+package duke;
+
+import duke.exceptions.*;
```

Comment:
You should import only the libraries that you are going to use explicitly instead of importing everything using the * wildcard. This makes your import libraries easier to comprehend and maintain.